receiptui/src/androidTest/java/com/hyperwallet/android/ui/receipt/ListPrepaidCardReceiptsTest.java

```diff
@@ -377,7 +404,9 @@ public void testListPrepaidCardReceipt_clickTransactionDisplaysDetails() {
         onView(withId(R.id.transaction_title)).check(matches(withText(R.string.prepaid_card_sale)));
         onView(withId(R.id.transaction_amount)).check(matches(withText("- 10.00")));
         onView(withId(R.id.transaction_currency)).check(matches(withText("USD")));
-        onView(withId(R.id.transaction_date)).check(matches(withText("June 6, 2019")));
+        onView(withId(R.id.transaction_date)).check(
+                matches(withText(dateConversion("June 6, 2019", "22:49:41", "MMMM d, yyyy",
```

Comment:
FYI: I can see the difference between time value and the mock json response file. i think it is not a major difference. But it a may produce test case fail. 
Could you please update this(Screen shot attached) code if your code works well?
<img width="1147" alt="Screenshot 2020-01-20 at 10 35 23 PM" src="https://user-images.githubusercontent.com/52773997/72745240-534f4c00-3bd5-11ea-9490-2a070b9961ec.png">

Reply:
updated the date and time.